pom.xml

```diff
@@ -4,7 +4,7 @@
     <groupId>su.litvak.chromecast</groupId>
     <artifactId>api-v2</artifactId>
     <version>0.9.2-SNAPSHOT</version>
-    <packaging>jar</packaging>
+    <packaging>bundle</packaging>
```

Comment:
> So, if you want it to be merged then please change it to the scheme where it updates the manifest as said in that link.

Sure, will do so, soon. It is your project.

> Neither I want the library users to spend their time for doing same investigation.

The user do not need to spent time to investigate this.
I assume you do also spent no time to investigate why the protobuf of google is using bundle packaging :wink: 
http://search.maven.org/#artifactdetails%7Ccom.google.protobuf%7Cprotobuf-java%7C2.6.0%7Cbundle

Will update this PR.

Thank you for your time.